### ISSUE TITLE
RemoteIO including old waila api; causing crashes in other mods

### ISSUE THREAD

**Issue by BBoldt** [s0cks/RemoteIO#12]
https://github.com/tterrag1098/WAILAPlugins/issues/17

:cry: 

```
<tterrag> go say "your WAILA API is outdated and you shouldn't even be shipping that API"
<tterrag> or just "ur doin it wrong"
<TehNut> "al ur shits fukt"
```

**Comment by s0cks** [comment]
I know I havent updated the api yet will do today

**Comment by s0cks** [comment]
Fixed in 2.3
